extensions/positron-python/src/client/interpreter/configuration/environmentTypeComparer.ts

```diff
@@ -391,13 +391,13 @@ export function getPyenvVersion(workspacePath: string | undefined): string | und
 
 /**
  * Check if a version is supported (i.e. >= the minimum supported version and < the maximum).
- * Also returns true if the version could not be determined.
+ * Returns false if the version could not be determined.
  */
 export function isVersionSupported(version: PythonVersion | undefined): boolean {
     return (
-        !version ||
-        (comparePythonVersionDescending(MINIMUM_PYTHON_VERSION, version) >= 0 &&
-            comparePythonVersionDescending(MAXIMUM_PYTHON_VERSION_EXCLUSIVE, version) < 0)
+        version !== undefined &&
```

Comment:
Do we want to return true for undefined versions? We may get less options in a dropdown, or it may be difficult to "force" an unsupported environment provider via supplying a path (or other consequences?). But I don't feel super strongly about this.

I'll wait to update tests on this decision ⏫